Source/WebCore/css/ShorthandSerializer.cpp

```diff
@@ -39,9 +39,9 @@
 #include "CSSValueList.h"
 #include "CSSValuePair.h"
 #include "CSSVariableReferenceValue.h"
-#include "ComputedStyleExtractor.h"
 #include "FontSelectionValueInlines.h"
 #include "Quad.h"
+#include "StyleExtractor.h"
```

Comment:
Can this use a forward declaration instead?

Reply:
I don't think so. This is the cpp file and it calls extractor.propertyValue() so it needs the full thing.